src/app/zap-templates/common/simulated-clusters/TestLogCommands.js

```diff
@@ -30,9 +30,15 @@ const Log = {
   response : { arguments : [] }
 };
 
+const Prompt = {
```

Comment:
Can you actually change the this label from Prompt to `UserPrompt` instead, like: 

``` 
const UserPrompt = {
  name : 'UserPrompt',
  arguments : [ { type : 'CHAR_STRING', name : 'message' } ],
  response : { arguments : [] }
};
```